tests/phpunit/integration/Modules/Analytics_4/SettingsTest.php

```diff
@@ -33,16 +34,85 @@ public function test_get_default() {
 				// TODO: These can be uncommented when Analytics and Analytics 4 modules are officially separated.
 				// 'accountID'       => '',
 				// 'adsConversionID' => '',
-				'propertyID'      => '',
-				'webDataStreamID' => '',
-				'measurementID'   => '',
-				'useSnippet'      => true,
-				'ownerID'         => 0,
+				'propertyID'           => '',
+				'webDataStreamID'      => '',
+				'measurementID'        => '',
+				'useSnippet'           => true,
+				'ownerID'              => 0,
+				'googleTagID'          => '',
+				'googleTagAccountID'   => '',
+				'googleTagContainerID' => '',
 			),
 			get_option( Settings::OPTION )
 		);
 	}
 
+	public function test_owner_id_is_set() {
+		$testcase = $this->get_testcase();
+		$settings = $this->get_setting_with_owned_keys();
+		$settings->register();
```

Comment:
It's usually important to `remove_all_actions|filters` for any relevant hooks that are set up within the register method before calling it to ensure we're the hooks are running with the right class instance, otherwise it can potentially result in confusing failures later. That might not be needed here (and this is probably copied from another test case) but worth mentioning 👍

Reply:
Makes sense. As this `SettingsTests` class extends the generic `SettingsTestCase` wich [already implements some cleanup functions](https://github.com/google/site-kit-wp/blob/e38d5331b8726986ed69cf0e715a06532c9a3709/tests/phpunit/includes/Modules/SettingsTestCase.php#L22-L40), I think all necessary cleanup is already implemented. Did I miss something?